cd/application-deployment/staging/vaec-celery-task-definition.json

```diff
@@ -68,6 +68,10 @@
           "name": "NOTIFICATION_FAILURE_REASON_ENABLED",
           "value": "True"
         },
+        {
+          "name": "VA_ONSITE_URL",
+          "value": "https://staging-api.va.gov"
```

Comment:
You've specified the Onsite staging environment for all of our environments.  Is that correct?

Reply:
Yeah, I'm actually going to update Prod to point to `https://api.va.gov` instead of staging. It looks like the "secrets" might need to be defined in their own environments though. 
Based on this message Kyle pointed out:

`
STOPPED (ResourceInitializationError: unable to pull secrets or registry auth: execution resource retrieval failed: unable to retrieve secrets from ssm: service call has been retried 1 time(s): AccessDeniedException: User: arn:aws-us-gov:sts::171875617347:assumed-role/project-dev-notification-api-task-execution-role/02256968181f4fb7a6432320a2c3fa3d is not authorized to perform: ssm:GetParameters on resource: arn:aws-us-gov:ssm:us-gov-west-1:171875617347:parameter/staging/notification-api/onsite/notification-priv because no identity-based policy allows the ssm:GetParameters action status code: 400, request id: 39f04c81-4c65-4d44-8576-a2f707b46919)
`

Reply:
I have changed Prod to `https://api.va.gov` and all other environments point to `https://staging-api.va.gov`